src/get-started/index.njk

```diff
@@ -0,0 +1,237 @@
+  
+  <ul class="govuk-list govuk-list--bullet">
+    <li><a href="https://github.com/dashouse/govuk-design-system-flow-diagrams" class="govuk-link">Flow diagrams for GOV.UK services</a></li>
+    <li><a href="https://github.com/clare-brown/govuk-designsystem-flow-diagram-mural" class="govuk-link">GOV.UK elements in Mural</a></li>
+    </ul>
+    </p>
+
+<h2 class="govuk-heading-l" id="High-fidelity-prototypes">
+ High-fidelity prototypes
+ </h2>
+ 
+ <p class="govuk-body">
+ High-fidelity prototypes are detailed, coded prototypes that mimic the behaviour of ‘finished’ services. High-fidelity prototypes will help you to perform user testing and demonstrate the service to stakeholders.
+ </p>
+ 
+ <p class="govuk-body">
+ HMRC’s preferred approach is to use the GOV.UK Prototype Kit. You will need a device, such as a MacBook, that isn’t managed by HMRC so you can install the necessary software. 
+ </p>
+ 
+ <p class="govuk-body">
+ You can <a href="https://design-system.service.gov.uk/get-started/prototyping/">follow instructions</a> in the GOV.UK Design System to get the kit running on your unmanaged device.
+ </p>
+ 
+ <h3 class="govuk-heading-m" id="Prototype-support">
+ Prototype support
+ </h3>
+ 
+ <p class="govuk-body">
+ The GOV.UK Prototype Kit is used across governments and there is a significant amount of community support available if you get stuck or need help. 
+ </p>
+ 
+ <p class="govuk-body">
+ Inside HMRC you can reach out in our #community-prototype Slack channel or speak to members of HMRC’s Design Resources Team. Support is also available on the cross-government Slack in the #govuk-prototype-kit channel.
+ </p>
+ 
+ <p class="govuk-body">
+ You can also find out more in our prototyping club. We have a range of videos covering common tasks, like getting set up, routing and session data available on <a href="https://drive.google.com/drive/folders/1k9ZkMXE3WPpNsU126ekggSO5rWZ4MusK?usp=sharing/" class="govuk-link">Google Drive</a>.
+ </p>
+ 
+ <h3 class="govuk-heading-m" id="Hosting-and-sharing-prototypes">
+ Hosting and sharing prototypes
+ </h3>
+ 
+ <p class="govuk-body">
+ You will need a GitHub account, which you can create, and a digital HMRC login which can be set up for you by your team lead. Once you have set up your account you can use the 'automatic prototype creation tool'. 
+ </p>
+
+ <p class="govuk-body">
+ If you are working on a new service or a new prototype follow the guidance on <a href="https://confluence.tools.tax.service.gov.uk/display/PROT/Creating+a+new+prototype+repository/" class="govuk-link">Confluence</a> to create a new prototype repository. If you are working on a service where a prototype already exists, speak to the senior interaction designer on your programme or reach out on HMRC Slack <a href="https://join.slack.com/share/enQtNzY0NjE3NjY5NDE4MS1lNDczNTQxODk5YzE4MDk0MGY5YTBkY2YwMDdiNWNmZDRhMDdkOGIxMDg3YjA0ZTg5OWUwYjZhMTNjNjFlNjc3" class="govuk-link">#community-prototype</a>.
```

Comment:
I don't think these links don't go where I'd expect them to go out of context. For '#community-prototype' could the link be expanded out to 'HMRC Slack #community-prototype'?